parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -305,6 +305,20 @@ else
         # -________________-
         systemctl restart --no-block apt-daily.service
         aptmarkWALinuxAgent unhold &
+    elif [[ $OS == $MARINER_OS_NAME ]]; then
+        {{- if EnableUnattendedUpgrade }}
```

Comment:
nitpick. move this else to outside of the os check?

Reply:
ah never mind ace has that download service outside of his {{- if EnableUnattendedUpgrade }}